iceoryx2/src/node/mod.rs

```diff
@@ -81,6 +81,33 @@
 //! # }
 //! ```
 //!
+//! ## Simple Event Loop With Disabled [`Signal`](iceoryx2_bb_posix::signal::Signal) Handling
+//!
+//! This example demonstrates how the [`Node`](crate::node::Node) can be used when another instance is handling
```

Comment:
It isn't immediately obvious what is meant by "instance". How about more generically (so another node, or something else entirely):

" ... when system signals are being handled elsewhere"